packages/flutter/lib/src/material/selectable_text.dart

```diff
@@ -580,7 +584,7 @@ class _SelectableTextState extends State<SelectableText> with AutomaticKeepAlive
   }
 
   @override
-  bool get wantKeepAlive => true;
```

Comment:
Should `updateKeepAlive` be called when focus changes?

Reply:
it is called in the editable_text base class